source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,176 @@
+Organizing Your Sidebar
```

Comment:
In preview mode, this page looks broken. Unsure if the file should be "md" instead of "rst" or what the issue may be.

Reply:
@amyblais I've built the docs locally and can confirm that this page is working as expected.